OptimizelySDKCore/OptimizelySDKCore/OPTLYDecisionService.m

```diff
@@ -487,14 +487,17 @@ - (BOOL)evaluateAudienceConditionsForExperiment:(OPTLYExperiment *)experiment
     // Log Experiment Evaluation Started
     NSString *logMessage = [NSString stringWithFormat:OPTLYLoggerMessagesAudienceEvaluatorEvaluationStartedForExperiment, experiment.experimentKey, [experiment getAudienceConditionsString]];
     [config.logger logMessage:logMessage withLevel:OptimizelyLogLevelDebug];
-    BOOL areAttributesValid = [[experiment evaluateConditionsWithAttributes:attributes projectConfig:config] boolValue];
+    NSNumber *areAttributesValid = [experiment evaluateConditionsWithAttributes:attributes projectConfig:config];
```

Comment:
Nit: I know it was like this before, but could we rename the variable from `areAttributesValid` to `result`? The former makes it sound like we're just validating the user attributes that were passed in.